config/manager/manager.yaml

```diff
@@ -45,12 +45,5 @@ spec:
             port: 8081
           initialDelaySeconds: 5
           periodSeconds: 10
-        resources:
-          limits:
-            cpu: 100m
-            memory: 30Mi
-          requests:
-            cpu: 100m
-            memory: 20Mi
```

Comment:
Why are you doing this? Apart from affecting our QoS class, certain environments (hi Dedicated!) require all our Pods to define explicit resource consumption.

Reply:
Actually, the pod was getting oomKilled because of which I just removed them. But now I have increased the limit.